x-pack/plugins/fleet/public/applications/fleet/sections/epm/screens/detail/index.test.tsx

```diff
@@ -137,9 +151,48 @@ describe('when on integration detail', () => {
       });
     });
   });
+
+  describe('and on the Policies Tab', () => {
+    const policiesTabURLPath = pagePathGetters.integration_details({ pkgkey, panel: 'policies' });
+    beforeEach(() => {
+      testRenderer.history.push(policiesTabURLPath);
+      render();
+    });
+
+    it('should display policies list', () => {
+      const table = renderResult.getByTestId('integrationPolicyTable');
+      expect(table).not.toBeNull();
+    });
+
+    it('should link to integration policy detail when an integration policy is clicked', async () => {
+      await mockedApi.waitForApi();
+      const firstPolicy = renderResult.getByTestId('integrationNameLink') as HTMLAnchorElement;
+      expect(firstPolicy.href).toEqual(
+        'http://localhost/mock/app/fleet#/integrations/edit-integration/e8a37031-2907-44f6-89d2-98bd493f60dc'
+      );
+    });
+  });
 });
 
-const mockApiCalls = (http: MockedFleetStartServices['http']) => {
+interface MockedApi {
+  /** Will return a promise that resolves when triggered APIs are complete */
+  waitForApi: () => Promise<void>;
+}
+
+const mockApiCalls = (http: MockedFleetStartServices['http']): MockedApi => {
```

Comment:
FYI: at some point, I would like to split this concept out into a reusable test utility allows you to compose "multiple mock api handlers" and still be able to "observe" they they are done processing.